src/jshint.js

```diff
@@ -4225,7 +4225,7 @@ var JSHINT = (function() {
     var i = 0;
     var inof = ["in", "of"];
     var level = 0; // BindingPattern "level" --- level 0 === no BindingPattern
-    var comma; // First comma punctuator at level 0
+    var firstcomma; // First comma punctuator at level 0
```

Comment:
> All I want is jshint to work again and I can update without repatching the comma fix in every version.

My apologies: until you wrote this, I didn't realize that JSHint was actually non-functional for you. Up until now, I thought you were interested in improving the readability of the source--that is why I have treated it as lower-priority than the work I have been doing recently. The fact that the change is necessary to fix a problem changes things a bit.

It also makes me curious, though: all of the project's unit tests are passing *without* this change (or your original change) in place. Can you explain more about the error you are experiencing (and how you are running JSHint itself)?